core/chains/evm/config/chaintype/chaintype.go

```diff
@@ -22,14 +22,15 @@ const (
 	ChainXLayer          ChainType = "xlayer"
 	ChainZkEvm           ChainType = "zkevm"
 	ChainZkSync          ChainType = "zksync"
+	ChainZircuit         ChainType = "zircuit"
 )
 
 // IsL2 returns true if this chain is a Layer 2 chain. Notably:
 //   - the block numbers used for log searching are different from calling block.number
 //   - gas bumping is not supported, since there is no tx mempool
 func (c ChainType) IsL2() bool {
 	switch c {
-	case ChainArbitrum, ChainMetis:
+	case ChainArbitrum, ChainMetis, ChainZircuit:
```

Comment:
do we want to include Zircuit here or not now

Reply:
I think you can skip adding it here since Zircuit was previously using the optimism chain type that isn't included here